.travis.yml

```diff
@@ -9,6 +9,7 @@ branches:
   only:
   - master
   - develop
+  - pypi
```

Comment:
Only for testing or should that stay? For what?

Reply:
yes, that's only for testing at the moment. the plan was to clean that up. there might also be some references to that branch from the readme.